crates/catalog-unity/src/datafusion.rs

```diff
@@ -228,7 +228,11 @@ impl SchemaProvider for UnitySchemaProvider {
                 let new_storage_opts = temp_creds.get_credentials().ok_or_else(|| {
                     DataFusionError::External(UnityCatalogError::MissingCredential.into())
                 })?;
-                let table = DeltaTableBuilder::from_uri(table.storage_location)
+                let table_url =
+                    deltalake_core::table::builder::ensure_table_uri(&table.storage_location)
```

Comment:
looking at how often we use this, maybe we should make this a root level export in `deltalake_core`?

Reply:
Yes, indeed!
I was actually working on it 😃